src/validate/mod.rs

```diff
@@ -15,7 +15,7 @@ lazy_static! {
         "^https?://[A-Za-z0-9-]+(?:\\.[A-Za-z0-9-]+)*(?::[0-9]+)?/(?:[A-Za-z0-9-]+/)*$"
     ).unwrap();
     static ref EMAIL_ADDRESS_FORMAT: Regex =
-        Regex::new("^[a-z0-9-]+@[a-z0-9-]+(?:\\.[a-z0-9-]+)+$").unwrap();
+        Regex::new("^[a-z0-9\\-!#\\$%\\&`\\*\\+/=\\?\\^{\\|}~]+@[a-z0-9-]+(?:\\.[a-z0-9-]+)+$").unwrap();
```

Comment:
We should probably port this over to the content-server and auth-server validation too.